nixos/tests/rundeck.nix

```diff
@@ -0,0 +1,122 @@
+import ./make-test-python.nix (
+  { pkgs, ... }:
+  {
+    name = "rundeck";
+
+    nodes = {
+      server =
+        { config, pkgs, ... }:
+        {
+          imports = [
+            ./rundeck.nix
+          ];
+
+          networking.firewall.allowedTCPPorts = [ 4440 ];
+
+          services.rundeck = {
+            enable = true;
+            serverHostname = "server";
+            adminUser = "testadmin";
+            adminPassword = "testpassword";
+
+            aclPolicies = {
+              "admin.aclpolicy" = ''
+                description: Admin ACL for tests
+                context:
+                  project: '.*'
+                for:
+                  resource:
+                    - allow: '*'
+                  job:
+                    - allow: '*'
+                  node:
+                    - allow: '*'
+                by:
+                  group: admin
+                ---
+                description: Admin ACL in application scope
+                context:
+                  application: 'rundeck'
+                for:
+                  resource:
+                    - allow: '*'
+                  project:
+                    - allow: '*'
+                by:
+                  group: admin
+              '';
+            };
+
+            database = {
+              type = "h2";
+            };
+          };
+
+          virtualisation.memorySize = 2048;
+          virtualisation.diskSize = 4096;
+        };
+
+      client =
+        { config, pkgs, ... }:
+        {
+          environment.systemPackages = with pkgs; [
+            curl
+            jq
+          ];
+        };
+    };
+
+    testScript = ''
+      import time
+
+      start_all()
+
+      server.wait_for_unit("rundeck.service")
+      server.succeed("systemctl status rundeck.service")
+
+      server.succeed("test -d /var/lib/rundeck")
```

Comment:
It is kinda silly to check those directories. They are indirectly checked below, too